tools/dependency_summary.py

```diff
@@ -213,6 +213,14 @@
             "fixup": "https://raw.githubusercontent.com/remram44/adler32-rs/master/LICENSE",
         }
     },
+    # This package is a part of a workspace, the workspace has the licensing file,
+    # But the crate itself does not, we link to the workspace's LICENSE
+    "tracing": {
```

Comment:
Added this manually since tracing is a part of a workspace aaand the published crate itself didn't have the license